src/mdio/builder/templates/seismic_2d_cdp.py

```diff
@@ -2,6 +2,9 @@
 
 from typing import Any
 
+from mdio.builder.schemas import compressors
```

Comment:
```suggestion
from mdio.builder.schemas.compressors import Blosc
```

Import the least amount possible -- Nit-pick

Reply:
we also use BloscCname from there too. pretty much just importing the namespace module.

`compressors.Blosc(cname=compressors.BloscCname.zstd)`